src/catalog/catalog_entry/schema_catalog_entry.cpp

```diff
@@ -34,6 +34,8 @@
 #include "duckdb/planner/constraints/bound_foreign_key_constraint.hpp"
 #include "duckdb/parser/constraints/foreign_key_constraint.hpp"
 #include "duckdb/catalog/catalog_entry/table_macro_catalog_entry.hpp"
+#include "duckdb/function/table/table_scan.hpp"
```

Comment:
Are these extra includes necessary here?

Reply:
I've removed:
```
#include "duckdb/function/table/table_scan.hpp"
#include "duckdb/planner/operator/logical_get.hpp"
```

Are there any more that seem questionable to you?